.github/workflows/authors-file.yml

```diff
@@ -0,0 +1,18 @@
+name: AUTHORS file
+
+on:
+  pull_request: { }
+
+jobs:
+  authors-file:
+    name: AUTHORS file
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout HEAD
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check whether ./AUTHORS is up-to-date
+        run: .github/workflows/authors-file.bash
```

Comment:
If you click on [details](https://github.com/Icinga/icinga2/actions/runs/6933055461/job/18858366742?pr=9911) for the failing job of this PR, the output is trying to be helpful but the helpful part is hidden somewhere in the middle.

So I'd either remove the debug output, or split this job into two parts: one that always succeeds and writes the helpful message in a file if there's a problem, and then a second job that outputs that file and fails if it exists/is non-empty.